lib/vrl/lookup/src/lookup_v2/mod.rs

```diff
@@ -1,51 +1,83 @@
-//! This crate contains all of the logic for lookup paths.
+//! This crate contains all of the logic for paths.
```

Comment:
Isn't this a module?

Reply:
I think as soon as the v1 dies, this gets hoisted up to the top

Reply:
Yes to both. It will eventually be a crate, but I'll rename it for now.